test/exercism/html_formatter_test.rb

```diff
@@ -37,11 +48,11 @@ def get_lexer(language)
   end
 
   def load_sample(name, language)
-    code = load_example name
-    @raw_lines = code.lines
+    @code = load_example name
+    @raw_lines = @code.lines
     lexer = get_lexer language
 
-    @lexemes = lexer.lex(code)
+    @lexemes = lexer.lex(@code)
     @output = @formatter.format(@lexemes)
 
     @doc = Loofah::HTML::DocumentFragment.parse(@output)
```

Comment:
I do have strong opinions on these :)

You can do whatever you like while you're developing, but when you check it in test code should be held to equal standard of code quality as the rest of the code. (This includes running linters like Rubocop on it, and paying attention to code complexity issues highlighted by tools like `flog`.)